### ISSUE TITLE
Loss can't converge very well and is unstable

### ISSUE THREAD

**Issue by yyx1107** [eriklindernoren/PyTorch-YOLOv3#217]
I put the VOC datasets'images and labels(txt) to the custom folder.
But when I train the data with darknet.53.conv.74 ,I found that the loss was always between 1.5 and 2.（Batchsize=8 ,lr=0.001.）
What led to this result?Thanks very much!!
![图片1](https://user-images.githubusercontent.com/50829614/60588789-75fd4d00-9dca-11e9-98ff-a84fb26fb6bb.png)

**Comment by yyx1107** [comment]
Hi@eriklindernoren ,can you help me with the issue?And What's your training results when you trained on 
coco datasets?

**Comment by LinglanZhao** [comment]
How did you solve it? I think the use of **tensor.data** in lines 166-169 in **models.py** cutted off gradient propagation.  @yyx1107

**Comment by YinlinHu** [comment]
You may check the loss after a longer training

**Comment by ZhxJia** [comment]
i have the same issue , have you solved this issue?

**Comment by EddieEduardo** [comment]
My loss cannnot converge，either，very high, like this : 300->200->300->400,  jump back and forth within this range , how did you  solve this problem ?
